src/test/java/seedu/address/logic/commands/SocialMediaCommandTest.java

```diff
@@ -43,8 +43,27 @@ public void execute_newSocialMedia_success() {
         Model expectedModel = new ModelManager(new AddressBook(model.getAddressBook()), new UserPrefs());
         expectedModel.setPerson(person, editedPerson);
         assertCommandSuccess(socialMediaCommand, model, expectedMessage, expectedModel);
+
     }
 
+    /*
```

Comment:
for this test case, should we delete it if it is not being used?

Reply:
yep